lib/money/currency.rb

```diff
@@ -50,6 +50,10 @@ def eql?(other)
       self.class == other.class && iso_code == other.iso_code
     end
 
+    def hash
+      [ self.class, iso_code ].hash
```

Comment:
Can't you get away with only dong `iso_code.hash`? Or do you expect currency to be extended in some places in the code?

Reply:
I think it should be an exact match with the definition of `eql?`, which includes the class.  If we ever do extend Currency, we don't want this to start failing.

Reply:
Yea, you're right it should be an exact match.